guides/guide-for-adding-windows-node.md

```diff
@@ -114,33 +269,32 @@ kubectl apply -f sig-windows-tools/kubeadm/flannel/kube-flannel-rbac.yml
 1. Install ContainerD, wins, kubelet, and kubeadm.
 
 ```PowerShell
-git clone https://github.com/kubernetes-sigs/sig-windows-tools
-cd .\sig-windows-tools\kubeadm\scripts\
-
 # Install ContainerD
```

Comment:
should we use https://github.com/microsoft/Windows-Containers/blob/Main/helpful_tools/Install-ContainerdRuntime/install-containerd-runtime.ps1 instead of the sig-windows scripts?

Reply:
I agree that leveraging the official scripts to deploy containerd is ideal. Plus, I had a comment above on which my recommendation is to stay away from cloning entire repos on the user environment. Totally fine to pull a specific file/script, but pulling a repo has two downsides: 1) It might leave unwanted files on the user environment and 2) requires the user to use GIT which is not a requirement to use Kubernetes.

Reply:
I like the idea of having a one way of installing containerd. It will avoid confusion in the long run. The install script on the MS repo is new but looks like we could leverage it (haven't tried it myself but it was based off a blog post I did).  The install-containerd script in this repo was a convivence since at the time there wasn't a ton of detail on how to do it. Since we now have the one from MS we could leverage that.  

Two concerns:
- being able to get changes from community merged quickly. I think myself and @marosset can help with that
- It installs nerdctl which isn't ready for more than basic commands and might cause more issues for folks that try to use it 

Maybe we can submit a pr to the MS script to optionally turn off nerdctl, which would test concern number one :-)

Note if we go that route, we should probably mark the one in this repo as deprecated in some way (there are some know issues with it anyways).  We also need to ensure that we don't install the CNI using the MS script.

@marosset thoughts?

Reply:
I also talked to @brasmith-ms that I generally would advocate to reduce the number of scripts for installation. We currently have a ton of different ones, and it just causes confusion on which is the newest one, etc.

I also talked to him about having a more clean installation path. As WinGet doesn't have support for Windows Server, we could also start with chocolatey packages. I would dream of something more similar to the linux way:

```powershell
# install container runtime and Windows service
choco install containerd
# install kubelet as a Windows service
choco install kubelet kubeadm kubectl

# ... kubeadm join on Windows node would automatically configure kubelet service  ...
# ... (kubeadm init on Linux node already installed hostprocess kube-proxy Windows pod) ...
# ... now we could install CNI plugin via hostprocess container by kubectl apply ...
```
Could we work on something like that instead of the scripts? I would be happy to help.

Reply:
I'm fine with either option, in fact if the chocolatey package turns out to be a valid option I can update the script in the MS repo to use this instead. The WindowsContainers repo is under the control of the Windows container product group at MS so I am not worried about that disappearing as long as Windows containers exist :)

Reply:
I don't think it should be a requirement for sys admins to install Chocolatey or scoop on their servers to set up Windows.  Anyone with that type of knowledge should be able to figure out how to tweak the deployment scripts to use those tools.  

When and if Winget is officially supported in WS I would be in favor of using.